src/lib/isAlpha.js

```diff
@@ -17,8 +17,14 @@ export default function isAlpha(_str, locale = 'en-US', options = {}) {
     }
   }
 
-  if (locale in alpha) {
-    return alpha[locale].test(str);
+  if (Array.isArray(locale)) {
+    const regex = locale.map(localeKey => alpha[localeKey])
+      .filter(localeRegex => !!localeRegex)
+      .join('|');
+
+    return new RegExp(`^(${regex})+$`, 'i').test(str);
+  } else if (locale in alpha) {
+    return new RegExp(`^${alpha[locale]}$`, 'i').test(str);
   }
```

Comment:
suggestion: You don't have to use `else if` if if you return anyway. Code is cleaner just using `if` statements.

```suggestion
  if (Array.isArray(locale)) {
    const regex = locale.map(localeKey => alpha[localeKey])
      .filter(localeRegex => !!localeRegex)
      .join('|');

    return new RegExp(`^(${regex})+$`, 'i').test(str);
  }
  
  if (!!alpha[localeKey]) {
    return new RegExp(`^${alpha[locale]}$`, 'i').test(str);
  }
```